src/cgame/cg_tutorial.cpp

```diff
@@ -444,13 +444,15 @@ static void CG_HumanText( char *text, playerState_t *ps )
 		}
 	}
 
-	if ( upgrade == UP_NONE ||
-	     ( upgrade > UP_NONE && BG_Upgrade( upgrade )->usable ) )
+	// Find next weapon in inventory.
+	weapon_t nextWeapon = CG_FindNextWeapon( ps );
```

Comment:
commits need to be reordered

Reply:
Are you sure? Beware that GitHub just makes garbage about commit ordering, it sometime lists them by date (and maybe the commit date that is not the author date) instead of the way they are chained together.

Reply:
Are you telling me it's actually impossible to see the order of the commits? Surely things cannot be so bad...

Reply:
I stopped to trust Github ordering of the commits long time ago.